src/test/scala/edu/ie3/simona/scheduler/ExtendedSimSchedulerSpec.scala

```diff
@@ -1,1049 +0,0 @@
-      eventQueue.poll(2, TimeUnit.SECONDS) match {
-        case event: CheckWindowPassed =>
-          val expectedHead = expectedElements.headOption.getOrElse(
-            fail("ExpectedElements is empty!")
-          ) match {
-            case expectedHead: CheckWindowPassed =>
-              expectedHead
-            case unexpectedEvent =>
-              fail(
-                s"Unexpected event when CheckWindowPassed was expected: $unexpectedEvent"
-              )
-          }
-          expectedElements.remove(0)
-          event.tick shouldBe expectedHead.tick
-
-        case event => fail(s"Expected Simulating event but received $event")
-      }
-
-      // return with completion incl. new trigger for resolution*4
-      expectReturnComplete(
-        scheduler,
-        Some(
-          Vector(
-            ScheduleTriggerMessage(
-              ActivityStartTrigger(resolution * 4),
-              testActor
-            )
-          )
-        )
-      )
-      expectReturnComplete(
-        scheduler,
-        Some(
-          Vector(
-            ScheduleTriggerMessage(
-              ActivityStartTrigger(resolution * 4),
-              testActor
-            )
-          )
-        )
-      )
-
-      // expect a checkWindowPassed for tick 2700, 3600 and a Done event
-      var runDone = false
-      while (!runDone) {
-        eventQueue.poll(2, TimeUnit.SECONDS) match {
-          case event: CheckWindowPassed =>
-            val expectedHead = expectedElements.headOption.getOrElse(
-              fail("ExpectedElements is empty!")
-            ) match {
-              case expectedHead: CheckWindowPassed =>
-                expectedHead
-              case unexpectedEvent =>
-                fail(
-                  s"Unexpected event when CheckWindowPassed was expected: $unexpectedEvent"
-                )
-            }
-            expectedElements.remove(0)
-            event.tick shouldBe expectedHead.tick
-
-            if (event.tick == 2700) {
-              // when we passed tick 2700 we also expect that two message arrive at tick 3600 we need to answer with complete
-              expectReturnComplete(scheduler)
-              expectReturnComplete(scheduler)
-
-              // we also expect that the scheduler answers with a SimulationSuccessfullMessage to the sender of the run request (this test)
-              expectMsgPF() {
-                case SimulationSuccessfulMessage =>
-                case msg =>
-                  fail(
-                    s"Unexpected message received when SimulationSuccessfulMessage was expected: $msg"
-                  )
-              }
-            }
-
-          case event: Done =>
-            runDone = expectDoneEvent(event, expectedElements)
-          case event => fail(s"Expected Simulating event but received $event")
-        }
-      }
-
-      expectedElements.size shouldBe 0
-
-    }
-
-    "finish simulation correctly when endTick < pauseScheduleAtTick" in {
```

Comment:
This does not seem to be tested in the newer implementation

Reply:
It is, here:

https://github.com/ie3-institute/simona/blob/63f862161c9efc09c54d0bc058b59e16c1d1555d/src/test/scala/edu/ie3/simona/scheduler/SimSchedulerSpec.scala#L347

Reply:
whoops